openmoc/plotter.py

```diff
@@ -321,12 +336,22 @@ def plot_cells(geometry, gridsize=250, xlim=None, ylim=None):
 
   if not is_integer(gridsize):
     py_printf('ERROR', 'Unable to plot the Cells since ' + \
-                'since the gridsize %d is not an integer', gridsize)
+                'the gridsize %d is not an integer', gridsize)
 
   if gridsize <= 0:
     py_printf('ERROR', 'Unable to plot the Cells ' + \
               'with a negative gridsize (%d)', gridsize)
 
+  if z_level is None:
```

Comment:
While going through this I've realized just how much redundant error checking code we have in these plotting routines. I recognize this isn't something that must be solved in this PR, but if you are going to change `z_level` to `zcoord` you might consider extracting and generalizing a lot of this error checking to some sort of (private) "openmoc.plotter._check_input(gridsize, xlim, ylim, zcoord)` routine.